src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftPluginDescriptor.java

```diff
@@ -37,5 +58,41 @@ default ListBoxModel doFillWaitUnitItems() {
         return items;
     }
 
+    default FormValidation doTestConnection(@QueryParameter String apiURL, @QueryParameter String authToken) {
 
+
+        if (StringUtils.isEmpty(apiURL)) {
```

Comment:
we should be able to fall back to seeing if "KUBERNETES_SERVICE_HOST" is set as an env var like the build steps do